glom/test/test_basic.py

```diff
@@ -200,6 +200,14 @@ def test_spec_and_recursion():
 def test_scope():
     assert glom(None, S['foo'], scope={'foo': 'bar'}) == 'bar'
 
+    target = range(3)
+    spec = [(S, lambda S: S['multiplier'] * S[T])]
```

Comment:
I think usage of `S` as a local variable here is going to encourage confusion between the global spec `S`, and the local `scope` argument to `glom`

I'd like to propose the following convention for our docs &etc: `lambda scope:` where possible; `lambda s:` where writing out `scope` is too obtrusive.  Aliasing a global is usually considered bad practice, and then consider that it is just 10 characters to the left being used to mean something completely different I think this is particularly dangerous.